packages/admin/src/emailCampaigns/statistics/PercentageCard.tsx

```diff
@@ -0,0 +1,147 @@
+import { Paper, Skeleton as MuiSkeleton, Typography } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import * as React from "react";
+import { FormattedNumber } from "react-intl";
+
+interface Props {
+    title: React.ReactNode;
+    currentNumber?: number;
+    targetNumber?: number;
+    variant?: "normal" | "circle";
+}
+
+export const PercentageCard: React.FC<Props> = ({ title, currentNumber, targetNumber, variant = "normal" }) => {
+    const percentage = currentNumber === undefined || targetNumber === undefined || targetNumber <= 0 ? null : (currentNumber / targetNumber) * 100;
+    const renderSkeleton = currentNumber === undefined || targetNumber === undefined;
+
+    const values = (
+        <>
+            <Typography variant="h1">
+                {renderSkeleton ? (
+                    <Skeleton variant="text" width={80} height={64} />
+                ) : (
+                    <>
+                        {percentage === null ? (
+                            "–"
+                        ) : (
+                            <>
+                                <FormattedNumber value={percentage} style="decimal" minimumSignificantDigits={3} maximumSignificantDigits={3} />%
+                            </>
```

Comment:
```suggestion
                            <FormattedNumber value={percentage} style="decimal" minimumSignificantDigits={3} maximumSignificantDigits={3} />%
```

Reply:
doesn't work,
there is a FormattedNumber + "%"